src/commands/utils/download/download.ts

```diff
@@ -1,16 +1,8 @@
 import * as path from 'path';
-import * as stream from 'stream';
 import { succeeded, Errorable, getErrorMessage} from '../errorable';
 import { Dictionary } from '../dictionary';
 import { sleep } from '../sleep';
-
```

Comment:
Note: this hack was originally added to avoid the `download` import overwriting the user's `HOME` environment variable. This happened because of a [transitive](https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools/pull/302#issuecomment-404678781) dependency on `npm-conf` via the [`caw`](https://www.npmjs.com/package/caw) package, which has since [been removed](https://github.com/kevva/download/commit/a16ba04b30dafbe7d9246db93f1534320d8e0dd3) as a dependency of `download`. I've verified that the `HOME` value is not being overwritten, even without this workaround.